### PR TITLE
Add License Meta

### PR DESCRIPTION
**Pull request by ciaranRoche**

### CODE REVIEW

File: licenses/licenses.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<?xml version='1.0'?>
+<licenseSummary>
+    <project>sync-android-app</project>
+    <version>0.0.1</version>
```
Comment (ciaranRoche) [inline licenses/licenses.xml]: @lgriffin Where will we find the version number of individual repo's?

Comment (lgriffin) [reply licenses/licenses.xml]: Take a look through some of the build files like the gradle files or the equiv of package.json if one exists. If no obvious version number exists just put in 0.0.0 or something similar. The version number is not a huge deal especially for templates

**Comment by lgriffin** [comment]
@ciaranRoche let me know when I can merge it

**Comment by ciaranRoche** [comment]
@lgriffin I changed the version number back to 0.0.0, feel free to merge if you are happy with it.

**Pull request merged**